formatter-junit/markdownlint-cli2-formatter-junit.js

```diff
@@ -38,6 +38,7 @@ const outputFormatter = (options, params) => {
   if (results.length === 0) {
     suite.
       testCase().
+      name(path.basename(__filename).replace(/\.js$/u, "")).
```

Comment:
Since this is used twice now, could you please capture it in a variable on line 14.5? Call it outputFormatterName, please.

Reply:
Done.